exporter/debugexporter/internal/normal/profiles.go

```diff
@@ -35,10 +36,15 @@ func (normalProfilesMarshaler) MarshalProfiles(pd pprofile.Profiles) ([]byte, er
 				buffer.WriteString(" samples=")
 				buffer.WriteString(strconv.Itoa(profile.Sample().Len()))
 
-				if profile.Attributes().Len() > 0 {
-					profileAttributes := writeAttributes(profile.Attributes())
+				if profile.AttributeIndices().Len() > 0 {
+					attrs := []string{}
+					for _, i := range profile.AttributeIndices().AsRaw() {
+						a := profile.AttributeTable().At(int(i))
+						attrs = append(attrs, fmt.Sprintf("%s=%s", a.Key(), a.Value().AsString()))
```

Comment:
What about values containing spaces? Better escape + quote it, maybe using `%q`.

Reply:
To keep things readable, this should happen in another PR.

Reply:
Hm, looks to me that this has been introduced by your changes. Another PR is easily forgotten without a tracking issue.

Reply:
No, we previously used the common `writeAttributes` method, which does the same thing.
https://github.com/open-telemetry/opentelemetry-collector/blob/4fc50a85ed71025010b18ceae315accba2bfa0ab/exporter/debugexporter/internal/normal/common.go#L15

Changing it there impacts all signals. Hence the new PR.